receiver/awsfirehosereceiver/README.md

```diff
@@ -62,3 +62,6 @@ See [documentation](https://docs.aws.amazon.com/firehose/latest/dev/create-desti
 The record type for the CloudWatch metric stream. Expects the format for the records to be JSON.
 See [documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Metric-Streams.html) for details.
 
+### otel_v1
+The OpenTelemetry 1.0.0 format for the CloudWatch metric stream.
```

Comment:
```suggestion
### otlp_v1
The OTLP v1 format as produced by CloudWatch metric streams.
```

Please carry this change throughout.  `otlpmetricstream` for the internal package name, etc.